lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -207,3 +235,26 @@ func loadAPNs(gateway *protos.Identity_Gateway) (map[string]*lte_models.ApnConfi
 
 	return apnsByName, apnResources, nil
 }
+
+// appendApnResourcesToSubProtos adds the gateway-specific apn resources data to subscriber protos before
+// returning to AGWs.
+func appendApnResourcesToSubProtos(subProtos []*lte_protos.SubscriberData, apnResources lte_models.ApnResources) []*lte_protos.SubscriberData {
+	for i := range subProtos {
+		subProto := subProtos[i]
+		if subProto.GetNon_3Gpp() == nil || subProto.Non_3Gpp.GetApnConfig() == nil {
```

Comment:
This is where the guarded getters are nice -- just do `subProto.GetNon_3Gpp().GetApnConfig()`